lib/pbench/server/api/__init__.py

```diff
@@ -186,22 +191,36 @@ def register_endpoints(api, app, config):
 
 
 def get_server_config() -> PbenchServerConfig:
+    """Get a pbench server configuration object
+
+    The file to use for the configuration is specifed by the environment
+    variable, `_PBENCH_SERVER_CONFIG`.
+
+    Raises:
+        ConfigFileNotSpecified : when no value is given for the environment
+            variable
+
+    Returns:
+        A PbenchServerConfig object
+    """
     cfg_name = os.environ.get("_PBENCH_SERVER_CONFIG")
     if not cfg_name:
         raise ConfigFileNotSpecified(
-            f"{__name__}: ERROR: No config file specified; set" " _PBENCH_SERVER_CONFIG"
+            f"{__name__}: ERROR: No config file specified; set _PBENCH_SERVER_CONFIG"
         )
 
-    try:
-        return PbenchServerConfig.create(cfg_name)
-    except BadConfig as e:
-        raise Exception(f"{__name__}: {e} (config file {cfg_name})").with_traceback(
-            e.__traceback__
-        )
+    return PbenchServerConfig.create(cfg_name)
+
+
+def create_app(server_config: PbenchServerConfig) -> Flask:
```

Comment:
NOTE: Webb's point earlier was that this method doesn't actually *return* anything; storing the `Flask` app where it can presumably be found (if that happens at all) is  a side-effect. There's no `return` statement, and therefore no `Returns` docstring nor return type hint!

Reply:
Line 247?